### PR TITLE
Hokusai updates #PLATFORM-1151

### PR DESCRIPTION
**Pull request by izakp**
Updates for current Hokusai (v0.5.2) and Kubernetes (v1.10.7) configuration and current best practices

See https://artsyproduct.atlassian.net/browse/PLATFORM-1151 for configuration that I've noticed needs updating across older applications

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -26,7 +25,11 @@ jobs:
           command: hokusai registry push --tag $CIRCLE_SHA1 --force --overwrite
       - run:
           name: Configure
-          command: hokusai configure --kubectl-version 1.6.3 --s3-bucket artsy-citadel --s3-key k8s/config --platform linux
+          command: hokusai configure --kubectl-version 1.10.7 --s3-bucket artsy-citadel --s3-key k8s/config --platform linux
+      - run:
+          name: Validate Kubernetes Yaml
```
Comment (dleve123) [inline .circleci/config.yml]: this is cool – is this something that you are prototyping in this project, and hoping to move to the default templates?

Comment (joeyAghion) [reply .circleci/config.yml]: +1

File: .dockerignore [hunk 2]
```diff
@@ -1,5 +1,8 @@
 .git
 .gitignore
+.dockerignore
```
Comment (joeyAghion) [inline .dockerignore]: inception!

File: hokusai/production.yml [hunk 3]
```diff
@@ -20,23 +20,45 @@ spec:
...
-        - configMapRef:
-            name: volley-environment
-        image: 585031190124.dkr.ecr.us-east-1.amazonaws.com/volley:production
-        imagePullPolicy: Always
-        name: volley-web
-        ports:
-        - containerPort: 8080
+        - env:
+          - name: NODE_ENV
+            value: production
```
Comment (joeyAghion) [inline hokusai/production.yml]: I tend to prefer these go into the config-map (so they're applied to everything consistently, including consoles), but that doesn't relate to this PR.

File: .circleci/config.yml (hunk 1 above)
Comment (izakp) [reply .circleci/config.yml]: @dleve123 yes pretty much as a step towards getting config updates integrated with CI

File: hokusai/production.yml (hunk 3 above)
Comment (izakp) [reply hokusai/production.yml]: Ah yes that was there already - it came up in the diff due to updating indentation of lists in the Yaml files.  Will update.

**Comment by joeyAghion** [comment]
Gonna merge but haven't applied anything...

**Comment by joeyAghion** [comment]
Failed [here](https://circleci.com/gh/artsy/volley/48) with:

```
#!/bin/sh -eo pipefail
kubectl --context staging apply -f ./hokusai/staging.yml --dry-run
error: error converting YAML to JSON: yaml: line 24: mapping values are not allowed in this context
Exited with code 1
```

**Comment by izakp** [comment]
@joeyAghion yey it works! :D https://github.com/artsy/volley/pull/7

**Pull request merged**